tbx/project_styleguide/templates/patterns/styleguide/components/components.yaml

```diff
@@ -130,6 +130,14 @@ tags:
     partner_logo max-100x90 format-webp loading="lazy" class="partners__logo":
       raw: |
         <img src="https://source.unsplash.com/100x90?logo" width="100" height="90" alt="" class="partners__logo">
+    value.image fill-525x510 format-webp saturation-0.6 as desktop_image:
```

Comment:
Could you add a comment here like `# event block` as the component yaml is so long?

Reply:
Added in 01ba3b7bf115fe2b9651aa60dc38f856bb83dbfc

Reply:
thank you!